azkaban-common/src/main/java/azkaban/container/models/AzKubernetesV1SpecBuilder.java

```diff
@@ -32,6 +32,7 @@
 import io.kubernetes.client.openapi.models.V1VolumeMount;
 import io.kubernetes.client.openapi.models.V1VolumeMountBuilder;
 import java.util.ArrayList;
+import java.util.HashSet;
```

Comment:
Could you remove this import since it is not used afterwards?

Reply:
Anyway to add this to the linter, so builds fail when unused imports are present?

Reply:
removing this for now, however all files in the project have some unused imports. Lets tackle that as a separate task.